Jenkinsfile

```diff
@@ -697,6 +697,7 @@ pipeline {
                             unitTestPost artifacts: ['nlt_logs/*'],
                                          testResults: 'nlt-junit.xml',
                                          always_script: 'ci/unit/test_nlt_post.sh',
+                                         referenceJobName: 'daos-stack/daos/release%252F2.2',
```

Comment:
@ashleypittman I added you for review of this line since it's added in the patch

Reply:
Yes, this change is expected and correct for the 2.2 branch.